### DISCUSSION TITLE
Welcome to earthdata-cloud-cookbook Discussions!

### DISCUSSION THREAD

**Discussion by erinmr** [NASA-Openscapes/earthdata-cloud-cookbook#192] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefaniebutland** [comment]
Hello all! I'm Stefanie Butland, on the core Openscapes Team. I co-lead Openscapes Champions Cohorts, teach lessons like Open Communities and our GitHub Clinic, support and learn alongside Openscapes Mentors, and document how we do these things, via Quarto books like this one, and the ([Openscapes Approach Guide](https://openscapes.github.io/approach-guide/)), and via managing projects on GitHub. 

I'm a [scientist](https://scholar.google.ca/citations?user=9HnWx2kAAAAJ&hl)-turned scientist-enabler through building communities based on trust and the [value of welcome](https://www.cscce.org/2017/07/13/the-value-of-welcome/).

**Comment by jules32** [comment]
Hi All! I'm Julie Lowndes, I lead Openscapes and co-lead the NASA Openscapes project with Erin Robinson. I am a marine ecologist working at the intersection of actionable environmental science, data science, and open science. My main daily tools are R, Quarto, GitHub, Google Drive, and Slack.
